.github/workflows/lint-yaml.yaml

```diff
@@ -0,0 +1,14 @@
+name: YAML lint
+
+on: [push, pull_request]
+
+jobs:
+  lintAllTheThings:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: yaml-lint
+        uses: ibiqlik/action-yamllint@v3
```

Comment:
How does this action work?
Does it perform the formatting?

Reply:
@danielezonca the action just performs a check, returning success or fail, it doesn't modify the YAML.
The formatting was done externally, so it would pass the checks.